desk/app/bait.hoon

```diff
@@ -184,7 +231,7 @@
         [~ ~]
       ?>  =('2' u.type)
       :-  `[%bite-2 token joiner metadata]
-      ?~  inviter-field=(~(get by fields.metadata) 'inviter')
+      ?~  inviter-field=(~(get by fields.metadata) 'inviterUserId')
```

Comment:
Why this change? afaict reel still sets `'inviter'`.

The evils of arbitrary/untyped metadata field names...

Reply:
Gross. Should've just erred on the side of not changing it.

But it is what it is, I suppose...